app/assets/javascripts/discourse/app/controllers/composer.js

```diff
@@ -546,8 +546,9 @@ export default Controller.extend({
       this.cancelComposer(differentDraftContext);
     },
 
-    save() {
-      this.save();
+    save(ignore, event) {
```

Comment:
we can't change this sadly, but this is a very bad name for action, `onSave` would be much better :/

It's also annoying we have an action and a fn named the same, because this kind of things will cause issue when we move to @action it would just be a regular fn, and we can't have:

```
@action
save() (}

save() {}
```

Reply:
maybe we rename / deprecate?